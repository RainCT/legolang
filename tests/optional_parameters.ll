def func(int x=0):
    print x

func(5)
func()

print ""

def mixed(int a, int b, int c=5):
    print a+b+c

mixed(1, 2, 3)
mixed(1, 2)
