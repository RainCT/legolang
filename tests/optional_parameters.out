5
0

6
8
